### DISCUSSION TITLE
Include Icon Set configuration support files in repo for easy access?

### DISCUSSION THREAD

**Discussion by hfloyd** [skartknet/Iconic#12] [Ideas]
I really love the versatility of this package, which of course, comes with some complexities related to configuring Icon sets. I know some of these complexities have been addressed on the main README, but perhaps it would be a good idea to include in the Repository a Folder for specific font-set configuration instructions/support files, etc? I think it would make it easier for others to contribute as they work out how to configure specific icon sets.

For instance:
```
/Configuration-Help/
-- /FontAwesome-5.3.0/
---- README.md (notes from Christopher's blog post, links to the font files he fixed up, etc.)
--/Glyphicons/
---- README.md (notes copied over from main Readme)
--/Material-Icons/
---- README.md (notes copied over from main Readme)
```

etc...

**Comment by skartknet** [comment]
You're totally right. Configuring the plugin is something that has always worried me because it's not easy. Any help to make it better is welcome!

**Reply by hfloyd** [reply]
Okay, I'll start putting something together for a PR.

I also think it might make sense to consolidate any troubleshooting discussions into this area, to keep the conversations and tips together.

**Reply by hfloyd** [reply]
PR: https://github.com/skartknet/Iconic/pull/14
